qualtran/surface_code/ccz2t_cost_model.py

```diff
@@ -152,6 +152,27 @@ def n_cycles(self, n_magic: AlgorithmSummary) -> int:
 
 
 def get_ccz2t_costs(
```

Comment:
This seems more general than CCZ2T, should we rename `get_physical_cost` and refactor into the PhysicalCost file?

Reply:
@NoureldinYosri has been working to generalize the infrastructure to support additional models (including the "Azure" model). What do you think, Nour?

Reply:
I think we should do it but not now. we currently have two cost models: CCZ2T and the azure model. but they do certain things differently. I don't think having one big method is a good idea.